app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -307,16 +314,9 @@ class BrowserFragment : WebFragment(), LifecycleObserver, View.OnClickListener,
         backButton = view.findViewById(R.id.back)
         backButton?.let { it.setOnClickListener(this) }
 
-        val blockIcon = view.findViewById<View>(R.id.block_image) as ImageView
-        blockIcon.setImageResource(R.drawable.ic_tracking_protection_disabled)
-
-        blockView = view.findViewById<View>(R.id.block) as FrameLayout
```

Comment:
Was there telemetry for this code that's being removed, that can also be removed?

Reply:
Somehow no there wasn't 👍